vllm/v1/serial_utils.py

```diff
@@ -140,7 +140,7 @@ def _encode_ndarray(
     ) -> tuple[str, tuple[int, ...], Union[int, memoryview]]:
         assert self.aux_buffers is not None
         # If the array is non-contiguous, we need to copy it first
-        arr_data = obj.data if obj.data.c_contiguous else obj.tobytes()
+        arr_data = obj.data if obj.flags.c_contiguous else obj.tobytes()
```

Comment:
`c_contiguous` can be directly accessed from the flags which tends to be a tiny bit faster: https://numpy.org/doc/stable/reference/generated/numpy.ndarray.flags.html